### DISCUSSION TITLE
How to set mariadb !!!!!

### DISCUSSION THREAD

**Discussion by xorgxx** [symfony/symfony#47305] [Q&A]
SF 5.0.5 php 7.3.11 mariadb-10.3.21

Juste not possible to connect to the database BY SYMFONY only, all "client" navicat, phpstorm acces wihtout any probleme !!!

in my env : 
```
DATABASE_URL=mysql://root:</98C3la>B<O3s6@localhost:3307/op5.dev?serverversion=mariadb-10.3.21
```

i have try most all .

thank

**Comment by boatware** [comment]
Hi, 
I've got a similiar (if not the same) problem, from CLI or with a simple php file creating an PDO object I can connect to the database from everywhere, just not from within symfony.

Could you please tell what your solution was?

I'm using symfony 6, with MariaDB v10.3.34 on the server. If you need any more information, let me know.

**Comment by wouterj** [comment]
Please use one of the channels described on https://symfony.com/support to get community support. The issue tracker should only be used to report bugs or discuss features.
